### DISCUSSION TITLE
Issue in vertical rotation

### DISCUSSION THREAD

**Discussion by hanuRakesh16** [yomotsu/camera-controls#474] [Q&A] [answered]
Hi all, I’m facing an issue in rotation. The object is rotating completely in horizontal , but in vertical (Up-down) it gets stopped. Someone help me to solve this issue.

Thanks in advance.

https://github.com/yomotsu/camera-controls/assets/57532278/fe51c9d0-2656-4a93-9893-3f0fad92f876

**Comment by yomotsu** [comment, marked as answer]
Because camera-controls is a sort OrbitControls, and limits the polar angle rotation.
If you wish that movement, try TrackballControls of three.js official repository.
https://threejs.org/examples/#misc_controls_trackball
